.github/workflows/regression-check.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         Prelude.Ahead
         Prelude.Async:12
         Prelude.Parallel
-        Prelude.Serial
```

Comment:
We need to add Data.Stream in this list, otherwise we will not be doing regression checks on it.